lib/__tests__/cli.test.js

```diff
@@ -12,6 +13,12 @@ const replaceBackslashes = require('../testUtils/replaceBackslashes');
 const fixturesPath = (...elems) => replaceBackslashes(path.join(__dirname, 'fixtures', ...elems));
 const { buildCLI } = cli;
 
+const removeFile = async (filePath) => {
+	if (existsSync(filePath)) {
+		await fs.unlink(filePath);
+	}
+};
```

Comment:
[suggest] We can now use the `testUtils/removeFile.js` helper. Can you please replace this code with the helper?

https://github.com/stylelint/stylelint/blob/005f39eea531283608afe5222e05a30c7d984d35/lib/testUtils/removeFile.js#L9

Reply:
Deleted cache test from `cli.test.js`. So removed `removeFile`. https://github.com/stylelint/stylelint/pull/6356/commits/fe3f4411257c48b14539480b4699f12a3f4a6a40